src/components/variant/drawer-tags.store.ts

```diff
@@ -0,0 +1,116 @@
+import { makeAutoObservable } from 'mobx'
+
+import { t } from '@i18n'
+import variantStore from '@store/variant'
+import { ITableAspectDescriptor } from '@service-providers/dataset-level/dataset-level.interface'
+import { findElementInRow } from '@utils/mian-table/find-element-in-row'
+import { showToast } from '@utils/notifications/showToast'
+import { noFirstSymbolsPattern } from '@utils/validation/validationPatterns'
+
+class DrawerTagsStore {
+  constructor() {
+    makeAutoObservable(this)
+  }
+
+  localCheckedTags: string[] = []
+  customTag: string = ''
+  errorMessage: string = ''
```

Comment:
properties and getter before the constructor
properties
...
getters
...
constructor
....
methods

Reply:
Fixed